contrib/pmsnare/pmsnare.c

```diff
@@ -1,38 +1,41 @@
 /* pmsnare.c
  *
- * this detects logs sent by Snare and cleans them up so that they can be processed by the normal parser
+ * This detects logs sent by Snare and cleans them up so that they can be processed by the normal parser.
  *
- * there are two variations of this, if the client is set to 'syslog' mode it sends
+ *
+ * There are two variations of this. If the client is set to 'syslog' mode, it sends:
  *
  * <pri>timestamp<sp>hostname<sp>tag<tab>otherstuff
  *
- * if the client is not set to syslog it sends
+ * If the client is not set to syslog, it sends:
  *
  * hostname<tab>tag<tab>otherstuff
  *
- * ToDo, take advantage of items in the message itself to set more friendly information
- * where the normal parser will find it by re-writing more of the message
+ * ToDo: Take advantage of items in the message itself to set more friendly information
+ * where the normal parser will find it by re-writing more of the message.
+ *
+ *
+ * Interesting information includes:
  *
- * Intereting information includes:
+ * In the case of Windows Snare messages:
+ *	the system hostname is field 12
+ *	the severity is field 3 (criticality ranging form 0 to 4)
+ *	the source of the log is field 4 and may be able to be mapped to facility
  *
- * in the case of windows snare messages:
- *   the system hostname is field 12
- *   the severity is field 3 (criticality ranging form 0 to 4)
- *   the source of the log is field 4 and may be able to be mapped to facility
- * 
- * 
- * created 2010-12-13 by David Lang based on pmlastmsg
+ *
+ * Created 2010-12-13 by David Lang based on pmlastmsg.
+ * Modified 2016-12-21 by Shane Lawrence.
  *
  * This file is part of rsyslog.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * 
- *       http://www.apache.org/licenses/LICENSE-2.0
- *       -or-
- *       see COPYING.ASL20 in the source distribution
- * 
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *		-or-
+ *		see COPYING.ASL20 in the source distribution
```

Comment:
I can't tell through the github interface, is this replacing a tab with spaces? The correct indentation here should be one tab.